aioesphomeapi/api.proto

```diff
@@ -1125,6 +1125,7 @@ enum MediaPlayerState {
   MEDIA_PLAYER_STATE_IDLE = 1;
   MEDIA_PLAYER_STATE_PLAYING = 2;
   MEDIA_PLAYER_STATE_PAUSED = 3;
+  MEDIA_PLAYER_STATE_ANNOUNCING = 4;
```

Comment:
As per my comment in the esphome pr. Please also bump the api client version in this library so that the device knows the client can accept this new state.

Reply:
I thought the the version is updated automatically by the CI, that's why I didn't change any yet. Can you give me a hint where I have to bump the api client version?

Reply:
https://github.com/esphome/aioesphomeapi/blob/e97a716d44ac8759dcdf497c693d66963806bd27/aioesphomeapi/connection.py#L148

Reply:
You can just bump the minor version to 11

Reply:
The API version and the library version are two different things =)

Reply:
ahh great, thanks for clarifying!